chaos_genius/utils/metadata_api_config.py

```diff
@@ -66,6 +67,13 @@
         "supported_aggregations": ["mean", "sum", "count"],
         "supports_multidim_dd": True
     },
+    "ClickHouse": {
+        "tables": True,
+        "views": False,
+        "materialized_views": False,
```

Comment:
Looks like views and materialized views are both supported by clickhouse: https://clickhouse.com/docs/en/sql-reference/statements/create/view/

We can set these to `True`

Reply:
Sure, here we can set them to True